src/Codec/Xlsx/Parser.hs

```diff
@@ -136,69 +156,65 @@ extractCellValue _ "b" "0" = [CellBool False]
 extractCellValue _ _ _ = []
 
 -- | Get xml cursor from the specified file inside the zip archive.
-xmlCursor :: Zip.Archive -> FilePath -> Maybe Cursor
-xmlCursor ar fname = parse <$> Zip.findEntryByPath fname ar
+xmlCursor :: Zip.Archive -> FilePath -> Parser (Maybe Cursor)
+xmlCursor ar fname = maybe (Right Nothing) parse $ Zip.findEntryByPath fname ar
```

Comment:
OK, so I'm merging it and will add follow up commits a bit later in the evening, thank you